### PR TITLE
WIP: Standardize package names

### PR DESCRIPTION
**Pull request by calvinchengx**
**Detailed description**:

Standard Java package names according to convention.  Java package names are not camelCase. They are small case.

### CODE REVIEW

File: src/main/java/com/hedera/databaseUtilities/DatabaseUtilities.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package com.hedera.databaseUtilities;
+package com.hedera.databaseutilities;
```
Comment (steven-sheehy) [inline src/main/java/com/hedera/databaseUtilities/DatabaseUtilities.java]: Let's consolidate the various util classes into a single package. Move this to `com.hedera.mirror.util`.

File: src/main/java/com/hedera/configLoader/ConfigLoader.java [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package com.hedera.configLoader;
+package com.hedera.configloader;
```
Comment (steven-sheehy) [inline src/main/java/com/hedera/configLoader/ConfigLoader.java]: Let's do `com.hedera.mirror.config` as we may have more config soon with Spring Boot.

**Comment by calvinchengx** [comment]
All packages have been nested under `com.hedera.mirror` as pointed out by @steven-sheehy 

This avoids namespace conflicts with other Hedera Java projects.

**Comment by steven-sheehy** [comment]
Can you fix the merge conflicts and update to com.hedera.mirror in non-java files?

**Comment by mike-burrage-hedera** [comment]
Going to close this PR soon as it's a month old with no activity unless there are objections.

**Comment by steven-sheehy** [comment]
I'd say close it. It's probably too hard anyway to fix the merge conflicts anyway. However, I would like to fix package names soonish.

**Pull request closed without merging**